pitaya-sharp/NPitaya/src/PitayaCluster.RPC.cs

```diff
@@ -46,16 +51,30 @@ private static async Task HandleIncomingRpc(IntPtr cRpcPtr)
 
                 Marshal.FreeHGlobal(res.data);
                 Marshal.FreeHGlobal(resPtr);
+                if (success)
+                {
+                    MetricsReporters.ReportTimer(Metrics.Constants.Status.success.ToString(), route,
```

Comment:
Hardcoded to "" and "PIT-500"? What about the actual status code returned by the server?

Reply:
Copied from the other spot doing the same... Need to be able to access this Info